build/codegen/rook-client-python/README.md

```diff
@@ -0,0 +1,67 @@
+# rook-client-python
+
+Automatically generated models for Rook's custom resource definitions.
+
+Right now, it supports three operators:
+
+* Ceph
+* Edgefs
+* Cassandra
+
+It is used to type check client code against the Rook API
+
+Inspired by https://github.com/kubernetes-client/python/tree/master/kubernetes/client/models
+
+Main uses case is the mgr/rook orchestrator module of the Ceph MGR
+
+
+## Example
+
+```python
+def objectstore(api_name, name, namespace, instances):
+    from rook_client.ceph import cephobjectstore as cos
+    rook_os = cos.CephObjectStore(
+        apiVersion=api_name,
+        metadata=dict(
+            name=name,
+            namespace=namespace
+        ),
+        spec=cos.Spec(
+            metadataPool=cos.MetadataPool(
+                failureDomain='host',
+                replicated=cos.Replicated(
+                    size=1
+                )
+            ),
+            dataPool=cos.DataPool(
+                failureDomain='osd',
+                replicated=cos.Replicated(
+                    size=1
+                )
+            ),
+            gateway=cos.Gateway(
+                type='s3',
+                port=80,
+                instances=instances
+            )
+        )
+    )
+    return rook_os.to_json()
+```
+
+## Demo
+
+![](rook-python-client-demo.gif)
+
+## Regenerate
+
+To re-generate the python files, run 
+
+```bash
+./generate.sh
```

Comment:
Should we document `make codegen` here as the recommended method instead of directly running the script?

Reply:
Hm. `build/codegen/python` has no dependency to Go etc. If you're a pure python developer, you don't need to call make to develop with the Pyhton API.